lib/get_navigation/src/extension_navigation.dart

```diff
@@ -600,10 +601,44 @@ extension GetNavigation on GetInterface {
   /// or also like this:
```

Comment:
I couldn't put the comment on the right line (598), so putting quite lower.

Would like to find out if this comment is outdated, since I can't make the same predicate work for `offUntil()` extensions. After debugging I found out that for this extension it makes no sense to check `currentRoute`, since `currentRoute` will always be the page where user is till the whole logic finishes.